### PR TITLE
Refactor ArgumentUtils into BundleUtils

### PR DESCRIPTION
**Pull request by lukstbit**
## Purpose / Description

In this PR:

 - refactor ArgumentUtils and its tests into BundleUtils and its tests
 - solve a small @KotlinCleanup in BundleUtilsTest
 - refactor a @KotlinCleanup for TemporaryModel as discussed in #11208

## How Has This Been Tested?

Ran the usual tests.

## Checklist

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/utils/BundleUtilsTest.kt [hunk 1]
```diff
@@ -16,13 +16,22 @@
...
 import org.mockito.Mockito.*
 import org.mockito.kotlin.whenever
-import java.util.*
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import kotlin.random.Random
 import kotlin.test.assertNull
 
+@RunWith(RobolectricTestRunner::class)
+@Config(application = EmptyApplication::class)
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/utils/BundleUtilsTest.kt:34]: These lines are a significant performance hit in unit tests.

Could you split the new tests out to `BundleUtilsRobolectricTest` instead.

**Review by david-allison — CHANGES_REQUESTED** [review]
I like it! One request for unit test performance.

File: AnkiDroid/src/test/java/com/ichi2/utils/BundleUtilsTest.kt (hunk 1 above)
Comment (lukstbit) [reply AnkiDroid/src/test/java/com/ichi2/utils/BundleUtilsTest.kt]: Done! I reverted a bit and used the previous ArgumentUtilsTest class and renamed it to BundleUtilsRobolectricTest.

Comment (david-allison) [reply AnkiDroid/src/test/java/com/ichi2/utils/BundleUtilsTest.kt]: I don't see a rename, only a 'delete'

Comment (lukstbit) [reply AnkiDroid/src/test/java/com/ichi2/utils/BundleUtilsTest.kt]: The deletion is ArgumentUtils from where I moved the method. ArgumentUtilsTest contains the two methods tested with Robolectric and this is the class I renamed.

**Review by david-allison — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
great, thank you!

**Pull request merged**